Homework/src/by/tms/Lesson_12/additionalTasks/task_3/Runner.java

```diff
@@ -0,0 +1,105 @@
+package by.tms.Lesson_12.additionalTasks.task_3;
+
+import java.io.BufferedReader;
+import java.io.FileNotFoundException;
+import java.io.FileReader;
+import java.io.IOException;
+import java.util.ArrayList;
+import java.util.List;
+
+public class Runner {
+    /*
+    3) Проверка на цензуру:
+    Создаете 2 файла.
+    1 - й. Содержит исходный текст.
+    2 - й. Содержит слова недопустимые в тексте(black list). Структура файла
+    определите сами, хотите каждое слово на новой строке, хотите через запятую
+    разделяйте, ваша программа делайте как считаете нужным.
+    Задача: необходимо проверить проходит ли текст цензуру. Если в тексте не
+    встретилось ни одного недопустимого слова, то выводите сообщение о том что
+    текст прошёл проверку на цензуру. Если нет, то выводите соответствуюущее
+    сообщение, кол-во предложений не прошедших проверку и сами предложения
+    подлежащие исправлению.
+     */
+    public static void main(String[] args) throws IOException {
+        BufferedReader reader = null;
+
+        String[] sentences = getSentences(reader);
+        if (sentences == null) {
+            System.err.println("There is no text file!");
+            return;
+        }
+
+        String[] blacklist = getBlackList(reader);
+        if (blacklist == null) {
+            System.err.println("There is no blacklist file!");
+            return;
+        }
+
+        checkCensure(sentences, blacklist);
+    }
+
+    public static String[] getSentences(BufferedReader reader) throws IOException {
+        try {
+            reader = new BufferedReader(new FileReader("/Users/trifonovanv/Desktop/TeachMeSkills/TMS_Homework/" +
```

Comment:
you should expect, that reader will not be null, or either create it here, or check if it is not null. And close it after using